### PR TITLE
Update readme to include openal-soft build information

### PR DESCRIPTION
**Pull request by ktadiko**
Just make sure it's working on your setups

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,18 @@ High level design decisions can also be documented here.
...
 
-Check OpenGL version `glxinfo | grep "OpenGL version"` (Linux)   
+* Check OpenGL version `glxinfo | grep "OpenGL version"` (Linux)
+    * Edit ` GLFW_CONTEXT_VERSION_MAJOR` and `GLFW_CONTEXT_VERSION_MINOR` in `engine/src/engine.cpp` to represent version on your computer.
+    * For example, if OpenGL version = 4.2, `GLFW_CONTEXT_VERSION_MAJOR = 4` and `GLFW_CONTEXT_VERSION_MINOR = 2`
 
-Edit ` GLFW_CONTEXT_VERSION_MAJOR` and `GLFW_CONTEXT_VERSION_MINOR` in `engine/src/engine.cpp` to represent version on your computer
-
-For example, if OpenGL version = 4.2, `GLFW_CONTEXT_VERSION_MAJOR = 4` and `GLFW_CONTEXT_VERSION_MINOR = 2`
+* Go to third_party/openal-soft/build and run `cmake ..`
```
Comment (hungrysushi) [inline README.md:20]: Can we do this in the top build directory? I don't see a reason to have a separate build directory in the submodule

**Comment by hungrysushi** [comment]
Works fine for me, but I'll let @atfreij approve once he's tested it

File: README.md (hunk 1 above)
Comment (ktadiko) [reply README.md:20]: This part was added to just to verify if the correct backends were found. It's not really needed unless there are issues like sound not being played.

File: third_party/CMakeLists.txt [hunk 2]
```diff
@@ -10,3 +10,7 @@ include_directories(glfw/include)
 
 # open gl hardware compatibility
 add_subdirectory(glad_generated)
+
+# OpenAL configuration
+add_subdirectory(openal-soft)
+include_directories(openal-soft/include)
```
Comment (hungrysushi) [inline third_party/CMakeLists.txt:16]: I think this is unnecessary. In the openal-soft [CMakeLists.txt](https://github.com/kcat/openal-soft/blob/master/CMakeLists.txt#L1303) it appears they export the includes

**Review by hungrysushi — APPROVED** [review]
I'm approving this. Up to you whether or not you want to update the third_party Cmakelists, but I'll let you decide either way and merge it.

File: third_party/CMakeLists.txt (hunk 2 above)
Comment (ktadiko) [reply third_party/CMakeLists.txt:16]: Leaving that in there for now so it doesn't look magical. I was wondering why it worked before.

**Pull request merged**